src/libraries/System.Speech/pkg/System.Speech.pkgproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Speech.csproj">
+      <SupportedFramework>netcoreapp2.0;net45;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Speech.csproj" />
+    <InboxOnTargetFramework Include="net45">
+      <AsFrameworkReference>true</AsFrameworkReference>
+    </InboxOnTargetFramework>
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>runtimes/win/lib/net45</TargetPath>
+    </File>
+    <PackageFile Include=".\build\System.Speech.targets">
```

Comment:
Can you do the following instead?   
```
build\netcoreapp2.0\System.Speech.targets
build\netcoreapp2.1\_._
```

This way you don't introduce targets into the supported case.

Also consider using `buildTransitive` instead of `build`.  https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files  Not sure if we can rely on that everywhere yet though 🤔 It's available with the 2.1 SDK https://docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-rtm

Reply:
re buildTransitive, since you didn't seem too sure, I left it as is for now. fixed the path.

Reply:
Submitted a PR to address Eric's remaining feedback: https://github.com/dotnet/runtime/pull/51925. Got to it as I'm currently doing something similar for System.Drawing.Common.